src/utils/gaa.js

```diff
@@ -1099,9 +1102,9 @@ export class GaaGoogle3pSignInButton {
   /**
    * Renders the third party Google Sign-In button for external authentication.
    * @nocollapse
-   * @param {{ allowedOrigins: !Array<string>, authorizationUrl: string }} params
+   * @param {{ allowedOrigins: !Array<string>, authorizationUrl: string, redirectMode: boolean }} params
    */
-  static show({allowedOrigins, authorizationUrl}) {
+  static show({allowedOrigins, authorizationUrl, redirectMode = true}) {
```

Comment:
Confirming that changing the default behavior is indeed desired? Also, can you please elaborate and document when you would want to set the flag to false?

Reply:
I'm really uncomfortable changing the default behavior of published code without a very clear analysis of why and if it will break existing users implementations.

Reply:
I've changed the redirectMode to false to avoid breaking any prior implementation. Added explanation in jsdoc.

My thinking around this was that we have not published GaaGoogle3pSignInButton in our DevSite yet and that the publishers who are implementing  GaaGoogle3pSignInButton are the ones that we work with directly so we could inform them of the change. But definitely not changing the default behavior is the safest approach. Thanks!